tests/test_lab_dev/test_states.py

```diff
@@ -87,6 +87,16 @@ def test_bargmann_triple_error(self):
         with pytest.raises(AttributeError):
             Number([0], n=10).bargmann
 
+    def test_normalize(self):
+        # Bargmann
+        coh = Coherent([0], x=1.0, y=1.0) * 0.5
+        normalized = coh.normalize()
+        assert np.isclose(normalized.probability, 1.0)
+        # Fock
+        coh.to_fock(5)  # truncated
```

Comment:
this doesn't do in-place modification, need to update `coh`
```suggestion
        coh = coh.to_fock(5)  # truncated
```